client/src/index.jsx

```diff
@@ -17,5 +17,5 @@ render((
     <Route path="/login" component={Login} />
     <Route path="/logout" component={Logout} onEnter={requireAuth} />
   </Router>
-  ), document.getElementById('app')
+  ), document.getElementById('app'),
```

Comment:
Could we override this rule to disallow dangling commas for functions:

```json
"comma-dangle": ["error", {
    "function": "never",
}]
```
? What do you think?